plugin/orderfield.go

```diff
@@ -0,0 +1,40 @@
+package plugin
+
+import (
+	"github.com/gogo/protobuf/protoc-gen-gogo/generator"
+)
+
+type OrmOrderField struct{}
```

Comment:
Any reason to have this struct? I would just have these functions to be methods of `*OrmPlugin`, like everywhere in the plugin.

Reply:
Hm..I will remove this struct.